### PR TITLE
Judge Dredd armor

### PR DESCRIPTION
**Pull request by NigglyWiggly**
Thanks to Skowron for the excellent sprites!
Warning: Do not give to sec.

image

### CODE REVIEW

File: code/modules/clothing/head/helmet.dm [hunk 1]
```diff
@@ -44,6 +44,13 @@
 	siemens_coefficient = 0.5
 	eyeprot = 1
 
+/obj/item/clothing/head/helmet/swat/dredd
+	name = "Judge Helmet"
+	desc = "Judge, Jury, and Executioner."
+	icon_state = "dredd-helmet"
+	item_state = "dredd-helmet"
+	species_fite = list()
```
Comment (DeityLink) [inline code/modules/clothing/head/helmet.dm]: >species_fite

It's "species_fit" blargh

Comment (NigglyWiggly) [inline code/modules/clothing/head/helmet.dm]: Fug

**Comment by DrCelt** [comment]
I AM

**Comment by DrCelt** [comment]
THE LAW

**Comment by Shadowmech88** [comment]
The right shoulderpad looks odd. Like he had a bite taken out of his arm.

**Comment by Probe1** [comment]
:+1:

**Pull request merged**